mkdocs.yml

```diff
@@ -9,7 +9,7 @@ copyright: "Copyright © 2022 Frequenz Energy-as-a-Service GmbH"
 repo_name: "frequenz-sdk-python"
 repo_url: "https://github.com/frequenz-floss/frequenz-sdk-python"
 edit_uri: "edit/v1.x.x/docs/"
-strict: true  # Treat warnings as errors
+# strict: true  # Treat warnings as errors
```

Comment:
This can be removed, https://github.com/marshmallow-code/marshmallow/issues/2739 is fixed.